Projeto-Stream/src/components/nav-bar.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div class="card z-10 fixed w-full ">
+        <prime-menubar :model="items" class="bg-black rounded-none">
+            <template #start>
+                <router-link :to="{name: 'home', query: {page:1}}">
+                    <img src="../assets/images/image.png" class="hidden md:block h-[40px]" alt="">
+                </router-link>
+            </template>
+            <template #item="{ item }">
+                <router-link :to="item.route">
+                    <span :class="item.icon" />
+                    <span class="!flex items-center ml-8 my-2 lg:mb-0 md:ml-6">{{ item.label }}</span>
+                </router-link>
+            </template>
+            <template #end>
+                <div class="flex align-items-center gap-2 mr-3">
+                    <router-link to="/favorites">
+                        <a href="/minha-lista"><svg stroke="currentColor" fill="red" stroke-width="0" viewBox="0 0 1024 1024" class="text-pirataflix-clean_red transform transition duration-150 ease-in-out hover:scale-105" height="1.5rem" width="1.5rem" xmlns="http://www.w3.org/2000/svg"><path d="M923 283.6a260.04 260.04 0 0 0-56.9-82.8 264.4 264.4 0 0 0-84-55.5A265.34 265.34 0 0 0 679.7 125c-49.3 0-97.4 13.5-139.2 39-10 6.1-19.5 12.8-28.5 20.1-9-7.3-18.5-14-28.5-20.1-41.8-25.5-89.9-39-139.2-39-35.5 0-69.9 6.8-102.4 20.3-31.4 13-59.7 31.7-84 55.5a258.44 258.44 0 0 0-56.9 82.8c-13.9 32.3-21 66.6-21 101.9 0 33.3 6.8 68 20.3 103.3 11.3 29.5 27.5 60.1 48.2 91 32.8 48.9 77.9 99.9 133.9 151.6 92.8 85.7 184.7 144.9 188.6 147.3l23.7 15.2c10.5 6.7 24 6.7 34.5 0l23.7-15.2c3.9-2.5 95.7-61.6 188.6-147.3 56-51.7 101.1-102.7 133.9-151.6 20.7-30.9 37-61.5 48.2-91 13.5-35.3 20.3-70 20.3-103.3.1-35.3-7-69.6-20.9-101.9z"></path></svg></a>
```

Comment:
Change this for <i class="pi pi-heart-fill " style="font-size: 2rem; color:red"></i>